src/liquid/components/contact_type_property.liquid

```diff
@@ -0,0 +1,44 @@
+{% if prop.type != 'generated' %}
+  {% capture prop_name %}{{ prefix }}{{prop.property_name}}{% endcapture %}
+
+  <div id="div_{{ prefix }}{{prop.property_name}}" class="field">
+    <label class="label">
+      {% if data.option_dob_unknown %}
+        Age in Years
+      {% else %}
+        {{ prop.friendly_name }}
+      {% endif %}
```

Comment:
Several files have lots of `if..else . Review the files to see: 
1. which ones can be made more readable by changing to case..when
2. Reducing cognitive complexity caused by deep nesting `if..else` inside` if..else` inside `for..in`
3. Some if else you could change to ternary operator
```suggestion
     {{ data.option_dob_unknown  ? 'Age in Years' : prop.friendly_name }}
```